py3status/i3status.py

```diff
@@ -143,6 +145,25 @@ def i3status_config_reader(self, i3status_config_path):
                 in_section = True
                 section_name = 'order'
 
+            if (not in_section) and line.startswith('group'):
```

Comment:
`not` comes before `and` in the order of operations, so these parentheses are unneeded. The same goes for several other places in this function.